### PR TITLE
Assign users to project

### PR DESCRIPTION
**Pull request by daothanhcam**

### CODE REVIEW

File: app/controllers/admin/projects_controller.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def create
     @project = Project.new project_params
     if @project.save
       flash[:success] = t "project.created"
-      redirect_to admin_projects_path
+      redirect_to  admin_project_members_path(@project)
```
Comment (khanhkhanh) [inline app/controllers/admin/projects_controller.rb]: 2 spaces after redirect_to

File: app/models/user.rb [hunk 2]
```diff
@@ -22,15 +24,27 @@ class User < ActiveRecord::Base
...
   scope :user_not_team, ->{where member}
   scope :select_leader, ->{where leader}
+  scope :no_project, ->{where member_not_in_project}
 
   def is_admin?
     role == Settings.user.role.admin
   end
 
+  class << self
+    def import(file)
```
Comment (dieunb) [inline app/models/user.rb]: remove `()`

File: app/models/user.rb [hunk 3]
```diff
@@ -22,15 +24,27 @@ class User < ActiveRecord::Base
...
+      CVS.foreach(file.path, header: true) do |row|
+        User.create!(row.to_hash)
```
Comment (dieunb) [inline app/models/user.rb]: remove `()`

File: config/routes.rb [hunk 4]
```diff
@@ -18,10 +18,14 @@
     resources :skills
     resources :positions
     resources :activity_logs, only: [:index, :destroy]
-    resources :projects
+    resources :projects do
+      resources :project_users
```
Comment (dieunb) [inline config/routes.rb]: `resource`

File: config/routes.rb [hunk 5]
```diff
@@ -18,10 +18,14 @@
...
+      get "members" => "project_users#show"
+    end
     resources :teams do
       resource :team_users
       get "members" => "team_users#show"
     end
+    collection {post :import}
```
Comment (dieunb) [inline config/routes.rb]: remove

**Pull request merged**